### PR TITLE
REM-680

### PR DESCRIPTION
**Pull request by RebeccaFrem02**
we need to introduce the 2nd FA that is an OTP form loaded if the user’s isEFAEnabled is true
SY.US.isEFAEnabled

### CODE REVIEW

File: src/components/Shared/OTPAuthentication.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import toast from 'react-hot-toast'
+import axios from 'axios'
+import { useAuth } from 'src/hooks/useAuth'
+import { useError } from 'src/error'
+import { ControlContext } from 'src/providers/ControlContext'
+import { AccessControlRepository } from 'src/repositories/AccessControlRepository'
+
+const OTPAuthentication = ({ loggedUser, onClose, window }) => {
+  const [otp, setOtp] = useState(['', '', '', '', '', ''])
+  const [error, setError] = useState('')
```
Comment (Alinadimmokdad) [inline src/components/Shared/OTPAuthentication.js:13]: this could be useState([])

Comment (RebeccaFrem02) [reply src/components/Shared/OTPAuthentication.js:13]: error is  string it is better if the initial value is ' '

File: src/components/Shared/OTPAuthentication.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+import styles from '../../../styles/phoneVerification.module.css'
+import toast from 'react-hot-toast'
+import axios from 'axios'
+import { useAuth } from 'src/hooks/useAuth'
+import { useError } from 'src/error'
+import { ControlContext } from 'src/providers/ControlContext'
+import { AccessControlRepository } from 'src/repositories/AccessControlRepository'
+
+const OTPAuthentication = ({ loggedUser, onClose, window }) => {
+  const [otp, setOtp] = useState(['', '', '', '', '', ''])
```
Comment (Alinadimmokdad) [inline src/components/Shared/OTPAuthentication.js:12]: this could be useState([]) ?

File: src/components/Shared/OTPAuthentication.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        })
+        .catch(error => {
+          showError({
+            message: error,
+            height: error.response?.status === 404 || error.response?.status === 500 ? 400 : ''
+          })
+          if (throwError) reject(error)
+        })
+    } else {
+      setError('All Fields Required')
```
Comment (FatimaSalami) [inline src/components/Shared/OTPAuthentication.js:57]: isn't it better to get the error msgs from labels?

**Review by FatimaSalami — APPROVED** [review]

**Pull request merged**